crates/libcontainer/src/process/fork.rs

```diff
@@ -289,9 +289,9 @@ mod test {
         match unsafe { unistd::fork()? } {
             unistd::ForkResult::Parent { child } => {
                 let sibling_process_pid =
-                    Pid::from_raw(receiver.recv().with_context(|| {
-                        "failed to receive the sibling pid from forked process"
-                    })?);
+                    Pid::from_raw(receiver.recv().with_context(
+                        || "failed to receive the sibling pid from forked process",
+                    )?);
```

Comment:
not necessarily related to rust update, but here because the value is 'static str, we can use `context` instead of `with_context` and get rid of the closure completely.